features/exploredash/src/main/res/layout/buy_gift_card_description.xml

```diff
@@ -74,25 +75,27 @@
         android:id="@+id/find_merchant_label"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        app:layout_constraintStart_toEndOf="@id/find_merchant_icon"
+        app:layout_constraintStart_toStartOf="@id/buy_gift_card_label"
         app:layout_constraintEnd_toEndOf="@id/right_guideline"
         app:layout_constraintTop_toTopOf="@id/find_merchant_icon"
         app:layout_constraintBottom_toBottomOf="@id/find_merchant_icon"
         android:text="@string/find_merchant_text"
         style="@style/Body2"
-        android:layout_marginLeft="20dp"
         android:paddingTop="16dp"
         android:paddingBottom="16dp"
         />
 
 
     <ImageView
         android:id="@+id/buy_gift_card_blue_icon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="34dp"
+        android:layout_height="25dp"
         app:layout_constraintTop_toBottomOf="@id/find_merchant_icon"
         app:layout_constraintStart_toStartOf="@id/left_guideline"
-        app:srcCompat="@drawable/ic_explore_info_screen_gift_card_blue"
+        android:src="@drawable/ic_gift_card"
```

Comment:
Removed some duplicated resources

Reply:
since we are not supporting Android 4.4 anymore, then `app:srcCompat` is not necessary?

Reply:
Correct, hence I'm not using it in the refactored code.